playbooks/startMasterWorker.yml

```diff
@@ -19,7 +19,7 @@
     command: screen -S H-Worker -dm ./runWorker.sh
     args:
       chdir: /home/ubuntu/HarmonicIO/
-    become: yes
+    become_user: ubuntu
```

Comment:
create the worker screens as root, so we can check them all (workers and masters) with a single remote SSH command

Reply:
isn't it a requirement to be root when starting the workers? or it were in the initial HIO setup, maybe it has changed?

Reply:
yes, I think its because it was trying to become 'root' and not 'ubuntu'. I guess the 'become user' should be set in the inventory file.

Reply:
Also 'become_user' means nothing unless 'become' set to true.
https://stackoverflow.com/questions/38290143/difference-between-become-and-become-user-in-ansible

I'll tidy this all